Cargo.toml

```diff
@@ -52,6 +52,8 @@ shuttle-actix-web = { version = "0.21.0", optional = true }
 shuttle-runtime = { version = "0.21.0", optional = true }
 shuttle-static-folder = { version = "0.21.0", optional = true }
 tokio = { version = "1.29.1", optional = true }
+serde_json = "1.0.100"
+chrono = "0.4.26"
```

Comment:
Let's remove the `chrono` dependency and deliver the UNIX timestamp instead of formatted time.

Reply:
I do understand where you are coming from, but if I am the admin of the instance (or someone who uploaded a bunch of files), I'd like to see when the files expire. I am not bad in mathematics, but I can't do that calculation quickly in my head.
Nor do I know off the top of my head whether I can use a bash function in jq to convert a field to another output, otherwise `date +'%Y-%m-%d %H:%M:%S' -d "@unixtimestamp"` would do the trick.

Maybe we could have 2 fields: e.g. `expiry_date` and `expiry_date_human` or something similar. 

Otherwise I would have to write a tool/wrapper to convert the unix timestamps. 

This is easy for people who use rpaste where a flag can be implemented to show human readable timestamps, but when someone uses curl, the unix timestamps are rather cumbersome.

Reply:
If the goal is to remove the dependency on chrono, I think it might be possible to do this using PrimitiveDateTime from the std time lib. 

I'll proceed with any approach you guys agree on.

Reply:
If it is possible to format the time without `chrono`, let's go with that then.

Reply:
May I ask why chrono is bad? Is it too big or does it have security issues? I am not trying to be difficult, I really don't know.

Reply:
There was a security issue for a long time and people have been switching to the `time` crate.

- https://rustsec.org/advisories/RUSTSEC-2020-0159.html
- https://stackoverflow.com/questions/73883691/the-latest-chrono-0-4-crate-uses-time-0-1-which-has-a-potential-segfault-how-t

Also, if it is just for formatting, there is no need to pull in a lot of dependencies I think.

Reply:
I did some research and it is nearly impossible to get a human readable date/time string with the Rust standard library.

Well, that's not entirely true. One has to write the code themselves. e.g. this is the [algorithm](http://git.musl-libc.org/cgit/musl/tree/src/time/__secs_to_tm.c?h=v0.9.15) that MUSL uses.

I can't believe this is not a part of Rust. One really does not need all the fancy stuff these libs provide. A simple function that returns a struct of 6 vars or just the vars themselves (year, month, day, hour, minute, second) would be enough. Maybe I should write a crate that does exactly that.

Reply:
I am seriously contemplating whether I should write a crate. It would have exactly one function which would take an i64 as an argument and return a struct with 6 members (Option<struct>).

Reply:
I am almost done. The only thing missing is the README and the doc comments.
I'll do that this evening and publish the crate.

Reply:
There you go: https://crates.io/crates/uts2ts

The docs need a while before they are generated, but I am pretty sure the example and README already explain everything.

Reply:
Nice work on this, it was very easy to implement.

Reply:
Wow, @tessus just dropped a new crate outta nowhere. Nice! 🫡

Reply:
Thanks guys! I was inspired by this situation. I really got annoyed that such a simple thing was missing from Rust. Pulling in `chrono` or `time` seemed like an overkill. 

I made sure that it will work for dates after 2038. With a 64bit implementation the date range is 584 billion years. I think that should do it. Which is why I did not bother to allow input of time units (ms, us, ns) other than seconds.

This crate does not even do the reverse. If something like that was needed, I'd write another crate `ts2uts` ;-) haha.

It gave me the opportunity to learn more about the Rust ecosystem, even though I still need to figure out a few things to improve my workflow. On one side there is a gh action workflow that does the deployment for me. But I still have to bump the version and change files manually. I saw that there are things like cargo-release and other tools, but I need them to work together. Testing all this stuff on a live project seems wrong.

If you know a nice hook manager, please let me know. There are a bunch of nice ones out there, but I haven't settled on one yet.

Btw, I stole your PR request template, but mentioned this most despicable deed in the commit message https://github.com/tessus/uts2ts/commit/0f604f9ee9b015fd056cd7d462f30e1a373ba556 ;-)

Reply:
I am thinking of changing the struct members (except year) to `u8`. This will not have any impact on anything, but the struct will allocate less memory. What do you think?